### DISCUSSION TITLE
elastic defend DB updates

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#12978] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

24

### RAM

128

### Storage for /

500GB

### Storage for /nsm

6TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,
i read in elastic websites about the elastic defend rules and db and how to update them in airgapped environments.
do i need to do this manually or should i wait for the next so release which will update them ?
tnx

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Looks like you can download and host the rules https://www.elastic.co/guide/en/security/current/offline-endpoint.html

**Reply by udi-mosh** [reply]
hi @reyesj2 ,
just to confirm, SO does not have a builtin solution for that, right ?
is so, would appreciate if an integrated solution will be applied.
